dosma/core/med_volume.py

```diff
@@ -646,12 +695,19 @@ def headers(self, flatten=False):
         Args:
             flatten (bool, optional): If ``True``, flattens header array
                 before returning.
+            json_dict (bool, optional): If ``True``, converts flattened headers to DICOM+JSON
+                Python dicts.
 
         Returns:
-            Optional[ndarray[pydicom.dataset.FileDataset]]: Array of headers (if they exist).
+            Optional[Union[ndarray[pydicom.dataset.FileDataset], Dict]]:
+                Array of headers (if they exist).
         """
         if flatten and self._headers is not None:
             return self._headers.flatten()
+
```

Comment:
it looks like `json_dict=True` depends on `flatten=True`? If so, we should enforce that or write in the comments that `json_dict=True` auto-flattens the array